### ISSUE TITLE
Safe Management to list the owners for all safes

### ISSUE THREAD

**Issue by Francesco-f7o** [cyberark/epv-api-scripts#311]
## Is your feature request related to a problem? Please describe.

As of today, SaaS customers don't have a way to easily produce a safe owners list using the report that used to be available in the PrivateArk client.
The entitlement report from the web portal is clumsy, listing the same owner multiple time on each safe, as well as incomplete, not including safes that don't contain passwords, e,g, recording safes.

## Describe the solution you would like

Combining the existing capabilities of the safe management script, where it can list all safes and detail the owners of a specific safe, we could produce a CSV file where we list the owners of all safes, providing what the PrivateArk Client used to do.
Basically repeating the retrieval of the owners for each safe we have permissions on, and redirecting the output into a CSV file.

## Describe alternatives you have considered

Some customers use the entitlement report from the web portal, others ask CyberArk support to run the report for them, and the support team manually runs and shares the report.

## Additional context

This is would be a win-win solution, for customers as they can get the reporting they need autonomously and in a faster way, and for CyberArk as we won't need to use support resources to manually execute reports,

**Comment by bab29** [comment]
This would need to be an enhancement request to CyberArk. Although limited reporting is available in some scripts it is not the primary focus of the scripts here.
